pkg/apis/discovery/types.go

```diff
@@ -116,6 +116,11 @@ type EndpointConditions struct {
 	// indicates an unknown state. In most cases consumers should interpret this
 	// unknown state as ready.
 	Ready *bool
+
+	// terminating indicates if this endpoint is terminating. Consumers should assume a
+	// nil value indicates the endpoint is not terminating.
```

Comment:
Maybe something closer to the comment for `ready` above?
```suggestion
	// terminating indicates if this endpoint is terminating. A nil value indicates an
	// unknown state. Consumers should interpret this unknown state indicates the
	// endpoint is not terminating.
```

Reply:
Will apply